V1/Services/CryptoService.cs

```diff
@@ -13,17 +15,20 @@ enum TransformDirection { ToCipher, FromCipher };
     public class CryptoService : ICryptoService
     {
         private readonly string serverKey;
+        private readonly RNGCryptoServiceProvider rng;
 
         public CryptoService(IConfiguration configuration)
         {
             serverKey = configuration.GetValue<string>("ServerKey");
+            rng = new RNGCryptoServiceProvider();
```

Comment:
The class is registered as a singleton, meaning the random number will be the same for everyone, for that instance.

Reply:
It'll be the same `RNGCryptoServiceProvider`, but that should be fine. Each time you call `rng.GetBytes()` you'll get a different random number.

Am I crazy?

Reply:
I think it should yes